test/integration/sanTokenTopTransactions.js

```diff
@@ -0,0 +1,49 @@
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHandlesNullData
+} = require('../integration_helper.js')
+
+const {
+  slug,
+  from,
+  to
+} = require('../setup.js')
+
+describe('SAN_TOKEN_TOP_TRANSACTIONS', () => {
+  const expected = {
+    date: 'string',
+    fromAddress: 'string',
+    fromAddressIsExchange: 'boolean',
+    toAddress: 'string',
+    toAddressIsExchange: 'boolean',
+    trxHash: 'string',
+    trxValue: 'number'
+  }
+
+  const response = san.SAN_TOKEN_TOP_TRANSACTIONS(slug, from, to, 5)
+  const headers = response[0]
+  const results = response[1]
+
+  testFieldTypes(results, expected)
+  testHandlesNullData(
+    'fetchTokenTopTransactions',
+    san.SAN_TOKEN_TOP_TRANSACTIONS,
+    slug,
+    from,
+    to,
+    5)
+
```

Comment:
Should we also have a test for `testHistoricDataIsForbidden`?

Reply:
No, I've tested the function in sanpy with dates from 2018 and it isn't a premium metric

Reply:
we have decided that it's not needed